plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/OpenShiftCoreMessages.java

```diff
@@ -20,6 +20,8 @@
     public static String OCBinaryLocationIncompatibleErrorMessage;
     public static String OCBinaryLocationDontExistsErrorMessage;
     public static String NoOCBinaryLocationErrorMessage;
+    public static String couldNoSyncPodsToFolder;
```

Comment:
I still believe that we should **NOT** extract strings, but if we do the correct naming would be could**Not**SyncPodsToFolder